integrations/azure-devops/azure_devops/client/azure_devops_client.py

```diff
@@ -91,15 +95,19 @@ async def generate_repository_policies(
     ) -> AsyncGenerator[list[dict[str, Any]], None]:
         async for repos in self.generate_repositories():
```

Comment:
Can we maybe pass the filter disables to the `generate_repositories` that way when we will have new resources that rely on repositories they will just have to pass the parameter ignore_disabled=True instead of adding a repeating if statement in their code?

Reply:
Cool done it